src/App.vue

```diff
@@ -212,6 +215,11 @@ export default {
     // add event listeners to track if network connection went up or down
     window.Offline.on("down", this.showInternetLostToast);
     window.Offline.on("up", this.showInternetRestoredToast);
+
+    // if user lands on home page, no transition should be applied
+    // else apply a transition with the name "slide-fade"
+    if (this.onHomePage) this.menuSlideTransition = "";
+    else this.menuSlideTransition = "slide-fade";
```

Comment:
only this case needs to be written here, the case for this.onHomePage being true suffices from being in the watcher itself